docs/source/docker/configuration/configuration.rst

```diff
@@ -51,8 +47,8 @@ The following are parameters which can be passed to the container:
   # Set to True to check whether installation was successful.
   sanity_check: False 
 
-  # Path to a file containing filenames to run the docker on a subset of the
-  # files in the input directory. The docker will ignore all files in the input 
+  # Path to a file containing filenames to run the Lightly Worker on a subset of the
+  # files in the input directory. The Lightly Worker will ignore all files in the input 
```

Comment:
input directory? maybe this comment can be a bit better structured with if datasource then else if input_dir